CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [0.27.0](https://github.com/awslabs/cdk8s/compare/v0.26.0...v0.27.0) (2020-08-05)
+
+
+### ⚠ BREAKING CHANGES
+
+* **lib:** resource names will now be rendered differently, omitting adjacent duplicate components.
+
+Signed-off-by: campionfellin <campionfellin@gmail.com>
+
+*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*
+
```

Comment:
Remove this

Reply:
"Signed-off-by" is also auto-generated when I [sign](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work) my commits.

I think the second part is part of the PR templates we have

Reply:
Yes but its usually not part of our release notes...not sure how it got here. lets remove it manually.

Reply:
Gotcha! Will remove it!